src/Button/Button.tsx

```diff
@@ -39,21 +38,39 @@ const Button: React.FC<ButtonProps> = ({
     title,
     large = false,
 }) => (
-    <button
-        type="button"
-        id={id}
-        className={classNames(
-            styles.button,
-            large && styles.large,
-            styles[variant],
-            className
-        )}
-        disabled={disabled}
-        onClick={onClick}
-        title={title}
-    >
-        {children}
-    </button>
+    <div className="tw-preflight">
+        <button
+            type="button"
+            id={id}
+            className={`${classNames(
+                large
+                    ? 'tw-h-8 tw-px-4 tw-text-sm'
+                    : 'tw-h-6 tw-px-2 tw-text-xs',
+                variant === 'primary' &&
+                    'tw-bg-nordicBlue tw-text-white active:enabled:tw-bg-nordicBlue-700',
+                variant === 'secondary' &&
+                    'tw-border tw-border-gray-700 tw-bg-white tw-text-gray-700  active:enabled:tw-bg-gray-50',
+                variant === 'success' &&
+                    'tw-bg-green tw-text-white  active:enabled:tw-bg-green-700',
+                variant === 'info' &&
+                    'tw-bg-nordicBlue tw-text-white active:enabled:tw-bg-nordicBlue-700',
+                variant === 'warning' &&
+                    'tw-bg-orange tw-text-white active:enabled:tw-bg-orange-700',
+                variant === 'danger' &&
+                    'tw-bg-red tw-text-white active:enabled:tw-bg-red-700',
+                variant === 'link' &&
+                    'tw-bg-transparent tw-p-0 tw-text-nordicBlue',
+                variant === 'link-button' &&
+                    'tw-border tw-border-nordicBlue tw-bg-white tw-text-nordicBlue active:enabled:tw-bg-gray-50',
+                className
+            )}`}
```

Comment:
Would be nice to have this logic extracted somewhere else to improve readability. It can be just a configuration object instead.

Reply:
If I take them out we loose the lint ordering and language server information at least that is my experience here why these are here

Reply:
Do you mean smth like this?
https://stackoverflow.com/questions/66614875/how-can-i-enable-tailwind-intellisense-outside-of-classname

Reply:
Problem is two fold:
1) if not associated with a var class/className we do not get language assistance like showing auto complete, showing the meaning of the class e.g. hovering on the class I get 

`.tw-text-white {
    --tw-text-opacity: 1 !important;
    color: rgb(255 255 255 / var(--tw-text-opacity)) !important;
}`

this is lost of not in a variable names class or className

We can add more regex but in the plugin settings but that is a developer thing and we cannot expect all of us to add multiple variables apart from the default as it will not be maintainable IMO

2) Even more problematic 

Formatting the order of the classes does not work if classnames are not a part of a an element inside ClassName property so solution for problem 1 will sill be a problem for this.

This helps sort all classes in the same order in out code and we cannot have this if classes are outside in some variable